### PR TITLE
chore: passing down ctx in the chain than placeholder TODO()

### PR DESCRIPTION
**Pull request by zdtsw**
we have context defined and pass from upper caller, this is just to make sure the ones we set to TODO can be using ctx

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: components/dashboard/dashboard.go [hunk 1]
```diff
@@ -276,7 +276,7 @@ func (d *Dashboard) deployConsoleLink(cli client.Client, owner metav1.Object, pl
 	pathConsoleLink := filepath.Join(manifestsPath, "consolelink.yaml")
 
 	consoleRoute := &routev1.Route{}
-	if err := cli.Get(context.TODO(), client.ObjectKey{Name: NameConsoleLink, Namespace: NamespaceConsoleLink}, consoleRoute); err != nil {
+	if err := cli.Get(context.Background(), client.ObjectKey{Name: NameConsoleLink, Namespace: NamespaceConsoleLink}, consoleRoute); err != nil {
```
Comment (ykaliuta) [inline components/dashboard/dashboard.go:279]: Why not here?

File: pkg/cluster/cluster_config.go [hunk 2]
```diff
@@ -17,11 +17,11 @@ import (
 
 // +kubebuilder:rbac:groups="config.openshift.io",resources=ingresses,verbs=get
 
-func GetDomain(c client.Client) (string, error) {
```
Comment (ykaliuta) [inline pkg/cluster/cluster_config.go:20]: Actually, unrelated change.

File: components/dashboard/dashboard.go (hunk 1 above)
Comment (zdtsw) [reply components/dashboard/dashboard.go]: missed it, updated

File: pkg/cluster/cluster_config.go (hunk 2 above)
Comment (zdtsw) [reply pkg/cluster/cluster_config.go]: reverted

**Review by ykaliuta — APPROVED** [review]
